config-jetson.ini

```
@@ -16,11 +21,20 @@ Host = 0.0.0.0
 QueuePort = 8010
 QueueAuthKey = shibalba
 
+[Source_0]
+VideoPath = /repo/data/softbio_vid.mp4
+Tags = area1
+Name = Garden-Camera
+Id = default
+Emails =
+NotifyEveryMinutes = 0
+ViolationThreshold = 60
+
 [Detector]
 ; Supported devices: Jetson , EdgeTPU, Dummy, x86
 Device = Jetson
-;  Name can be ssd_mobilenet_v2_coco which is trained on coco, or 
-; ssd_mobilenet_v2_pedestrian trained by neuralet on softbio dataset
+;  Name can be ssd_mobilenet_v2_coco which is trained on coco, or
+; ssd_mobilenet_v2_pedestrian_softbio trained by neuralet on softbio dataset
```

```suggestion
; ssd_mobilenet_v2_coco trained by neuralet on softbio dataset
```
as we don't have softbio model for the Jetson TX2 yet.

---

You are just changing the comment there. I guess we could instead add a note that that model is available only on Nano. And we could set the default value to the coco model

---

sorry, wrong line, I wanted to suggest on name key.